src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -8,10 +8,8 @@
     </None>
   </ItemGroup>
 
-  <!-- MSBuild for .NET Core doesn't support XslTransform at this time -->
   <Target Name="CompileAsn" BeforeTargets="CoreCompile"
-    Condition="'$(MSBuildRuntimeType)' != 'core'"
-    Inputs="@(AsnXml);$(MSBuildThisFileDirectory)asn.xslt"
```

Comment:
Removing `;$(MSBuildThisFileDirectory)asn.xslt` seemed like the right thing to do? With it there, the `Target` would run even if `@(AsnXml)` was empty, and the `XslTransformation` task would fail saying it wasn't given any input.

Reply:
Well, if the xslt isn't listed as an input then changing the xslt doesn't invalidate the target.  The fix is (I think) the above suggestion.

Reply:
Ah, okay. Got it.